src/module/addRemove.js

```diff
@@ -0,0 +1,178 @@
+    todoCheckboxElement.setAttribute('type', 'checkbox');
+    todoCheckboxElement.setAttribute('name', 'checkbox');
+    todoCheckboxElement.setAttribute('value', todoList[i].index);
+    todoCheckboxElement.checked = todoList[i].completed;
+
+    const todoDescriptionElement = document.createElement('p');
+    todoDescriptionElement.innerText = todoList[i].description;
+
+    todoCheckboxElement.addEventListener('change', () => {
+      if (todoCheckboxElement.checked) {
+        todoDescriptionElement.classList.add('strike');
+      } else todoDescriptionElement.classList.remove('strike');
+      toggleToDoStatus(todoList[i]);
+    });
+
+    const actionBtns = document.createElement('div');
+    actionBtns.classList.add('action-btn');
+    const editBtn = document.createElement('button');
+    editBtn.classList.add('edit-btn');
+    editBtn.classList.add('hide');
+    editBtn.setAttribute('type', 'button');
+    editBtn.innerHTML = '<i class="fa fa-edit"></i>';
+
+    editBtn.addEventListener('click', () => {
+      editTodoList(todoList[i]);
+    });
+
+    const deleteBtn = document.createElement('button');
+    deleteBtn.classList.add('delete-btn');
+    deleteBtn.classList.add('hide');
+    deleteBtn.setAttribute('type', 'button');
+    deleteBtn.innerHTML = '<i class="fa fa-trash">';
+
+    deleteBtn.addEventListener('click', () => {
+      removeTodoList(todoList[i].index);
+    });
+
+    const moreEllipsisBtn = document.createElement('button');
+    moreEllipsisBtn.classList.add('more-btn');
+    moreEllipsisBtn.setAttribute('type', 'button');
+    moreEllipsisBtn.innerHTML = '<i class="fa fa-ellipsis-v"></i>';
+
+    moreEllipsisBtn.addEventListener('click', () => {
+      editBtn.classList.toggle('hide');
+      deleteBtn.classList.toggle('hide');
+    });
+
+    todoContentElement.appendChild(todoCheckboxElement);
+    todoContentElement.appendChild(todoDescriptionElement);
+
+    actionBtns.appendChild(editBtn);
+    actionBtns.appendChild(deleteBtn);
+    actionBtns.appendChild(moreEllipsisBtn);
+
+    todoLiElement.appendChild(todoContentElement);
+    todoLiElement.appendChild(actionBtns);
+    todoListElement.appendChild(todoLiElement);
+  }
+  saveData();
+};
+
+const addTodo = () => {
+  const desc = document.querySelector('.taskList');
+  if (desc.value) {
+    const completed = false;
+    const description = desc.value;
+    const index = todoList.length + 1;
+    todoList.push({ completed, description, index });
+    displayToDo();
+    saveData();
+    desc.value = null;
+  }
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
+};
+
+const saveEdit = () => {
+  const desc = document.querySelector('.taskList');
+  if (desc.value) {
+    todoList = todoList.map((todo) => {
+      if (todo.index === todoEdit.index) {
+        return { ...todo, description: desc.value };
```

Comment:
This code is not clear, please create a variable and assign to it the value to return in this method.